app/controllers/users_controller.rb

```diff
@@ -1,6 +1,5 @@
 class UsersController < ApplicationController
   def index
-
     @users = User.order(:name)
   end
 end
```

Comment:
- [ ] Another issue I encountered while testing your app: there is no action method `show` in your users_controller. You have already used an instance variable `@user` in the users/show view which is not defined yet. Please try to create a show method with the instance variable `@user` so you'll be able to use it in the users/show view.

users/show.html.erb:
<img width="554" alt="image" src="https://user-images.githubusercontent.com/77190203/190605821-5cfde6a7-885d-4a9e-b309-3ebd3bf97c14.png">

Please ensure your app works as expected so we are able to test the authorization. The user should be able to create posts, comments, and likes. The ability to navigate across pages should function as expected. There should be no errors in your app.